### PR TITLE
chore: add a check arg to the migration command

### PR DESCRIPTION
**Pull request by aqeelat**
I also fixed a type and improved the styling of the command output.

### CODE REVIEW

File: auditlog/management/commands/auditlogmigratejson.py [hunk 1]
```diff
@@ -72,17 +88,25 @@ def _apply_django_migration(_logs) -> int:
...
-                    )
+                    errors.append(log.id)
                 else:
                     updated.append(log)
 
             LogEntry.objects.bulk_update(updated, fields=["changes"])
+            if errors:
+                self.stderr.write(
+                    self.style.ERROR(
+                        f"ValueError was raised while converting the the logs with these ids into json."
```
Comment (aleh-rymasheuski) [inline auditlog/management/commands/auditlogmigratejson.py:104]: Typo: `the the logs`

Comment (aqeelat) [reply auditlog/management/commands/auditlogmigratejson.py]: fixed

Comment (aqeelat) [reply auditlog/management/commands/auditlogmigratejson.py]: @aleh-rymasheuski sorry I forgot to say thank you. I've been under a lot of stress lately.

File: auditlog/management/commands/auditlogmigratejson.py [hunk 2]
```diff
@@ -1,28 +1,38 @@
...
+            help="Just check the status of the migration",
+            dest="check",
+        )
+        group.add_argument(
             "-d",
             "--database",
             default=None,
+            metavar="The database engine",
             help="If provided, the script will use native db operations. "
             "Otherwise, it will use LogEntry.objects.bulk_create",
```
Comment (aleh-rymasheuski) [inline auditlog/management/commands/auditlogmigratejson.py:28]: `bulk_update`, not `bulk_create`.

**Review by aleh-rymasheuski — DISMISSED** [review]
Only a typo in the existing code.

**Review by aleh-rymasheuski — APPROVED** [review]

**Review by hramezani — APPROVED** [review]

**Comment by hramezani** [comment]
Thanks @aqeelat

**Pull request merged**